### ISSUE TITLE
Manuscript style variation: needs background image support

### ISSUE THREAD

**Issue by justintadlock** [x3p0-dev/x3p0-ideas#3]
Placeholder ticket to build out the Manuscript style variation idea once background image support lands in Core: https://github.com/WordPress/gutenberg/pull/59354

**Comment by justintadlock** [comment]
This feature was added to Gutenberg 17.9, but theme-relative paths were not. There is not currently a specific ticket for relative paths, but they are in the tracking ticket and should get a followup before WP 6.6: https://github.com/WordPress/gutenberg/issues/54336

**Comment by justintadlock** [comment]
New PR for theme-relative paths here: https://github.com/WordPress/gutenberg/pull/60578

**Comment by ramonjd** [comment]
> There is not currently a specific ticket for relative paths, but they are in the tracking ticket and should get a followup before WP 6.6


Tracking ticket: https://core.trac.wordpress.org/ticket/61273

Backport PR is done and expected to land this week. Thanks!
